arabic/unpacked/math.js

```diff
@@ -0,0 +1,62 @@
+MathJax.Hub.Config({
+  Arabic: {
+    identifiersMap: {
+      // Math functions
+      'sin': 'جا',
+      'cos': 'جتا',
+      'tan': 'ظا',
+      'log': 'لو'
+    }
+  }
+});
+
+
+MathJax.Hub.Config({
+  Arabic: {
+    // Limits
+    operatorsMap: {
+      'lim': 'نهــا'
+    }
+  }
+});
+
+
+MathJax.Hub.Register.StartupHook('Arabic TeX Startup', function () {
+  var TeX = MathJax.Arabic.TeX;
+  var Text = MathJax.Arabic.Text;
```

Comment:
If you end up making an extension in `MathJax.Extension.Arabic`, as I recommended, then you will probably want to use that here rather than `MathJax.Arabic`.